clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/ClusteredMapRepository.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.client.internal.service;
+
+import org.ehcache.impl.internal.concurrent.ConcurrentHashMap;
+import org.terracotta.connection.entity.EntityRef;
+import org.terracotta.entity.map.common.ConcurrentClusteredMap;
+
+import java.util.concurrent.ConcurrentMap;
+
+class ClusteredMapRepository {
+
+  private final ConcurrentMap<String, Tuple<EntityRef<ConcurrentClusteredMap, Object>, ConcurrentClusteredMap>> knownMaps;
```

Comment:
Why do we need to keep the EntityRef around here?  It doesn't look like it's being used.